src/MSALWrapper/AuthFlow/Web.cs

```diff
@@ -117,8 +104,6 @@ protected override async Task<(TokenResult, IList<Exception>)> GetTokenInnerAsyn
                         .GetTokenInteractiveAsync(this.scopes, ex.Claims, cancellationToken),
                         this.errors)
                         .ConfigureAwait(false);
-
-                    return (tokenResult, this.errors);
                 }
             }
             catch (MsalServiceException ex)

```

Comment:
Can we collapse these catch statements into a single one?

Reply:
I think I did not fully understand what you are proposing. We are catching different exceptions to add to the error list. Please elaborate. :)

Reply:
This will be handled in one of the following PRs (it's already done)